partner_priority/models/partner_priority.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 Patrick Wilson <patrickraymondwilson@gmail.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models, fields, api
+
+
+class PartnerPriority(models.Model):
+    _name = 'partner.priority'
+    _order = 'sequence'
+    _description = 'Partner Priority'
+
+    name = fields.Char(string="Priority", required=True)
+    description = fields.Text(string="Description")
+    sequence = fields.Integer(string="Sequence")
+
+    @api.model
+    def create(self, vals):

```

Comment:
Why not a default in the field, to avoid inheriting the create method

Reply:
I'm not sure if I understand as the create is inherited for the sequence and i'm not setting any defaults. I'm not sure how to do that any other way actually.

Reply:
I have to agree with @luistorresm here. Using the sequence to get the next level of priority seems overly compicated. What if somebody wants to put in an intermediate priority? Would seem better to leave out the create method and the sequence. Just put default=1 (or 3), anyway the figure that stands for Normal as default.

Reply:
It might also be a good idea to make the sequence unique. Otherwise you might have an ambiguous priority.

Reply:
Or maybe like [here](https://github.com/odoo/odoo/blob/11.0/addons/stock/models/stock_quant.py#L336)

Reply:
Excellent! I see what you mean now. I didn't know this was a possibility and I like it. I'll be building all my sequences like this going forward. Thank you. This has been changed.